lib/mesh.js

```diff
@@ -215,30 +261,35 @@ Mesh.prototype.initialize = Promise.promisify(function(config, callback){
   }
 
   Object.defineProperty(this, 'version', {
-    value: JSON.parse(readFileSync(__dirname + '/../package.json')).version,
+    value: require(__dirname + '/../package.json').version,
     enumerable: true,
-  })
+  });
 
   // First mesh in process configures logger
   if (!Logger.configured) Logger.configure(config.util);
 
-  this.log = Logger.createLogger('Mesh');
+  _this.log = Logger.createLogger('Mesh');
+
   this.log.context = config.name;
```

Comment:
it's confusing to be using this and _this interchangeably. Is it on purpose (may need some clarifying comment) or can it be _this throughout?